snapcraft/models/project.py

```diff
@@ -1144,14 +1144,6 @@ def _validate_grade_and_build_base(self) -> Self:
             raise ValueError("grade must be 'devel' when build-base is 'devel'")
         return self
 
-    @pydantic.field_validator("build_base")
-    @classmethod
-    def _validate_build_base(
-        cls, value: str | None, info: pydantic.ValidationInfo
-    ) -> str | None:
-        """Build-base defaults to the base value if not specified."""
-        return value or info.data.get("base")
-
```

Comment:
Heck yeah.  This should be safe to remove and should be removed, so thanks for yanking off the bandage!